lib/protocol/constants/errors.js

```diff
@@ -1239,6 +1303,494 @@ exports.ER_XA_REPLICATION_FILTERS
+exports.ER_COLUMN_CHANGE_SIZE                                                            = 3886;
+exports.ER_REGEXP_INVALID_CAPTURE_GROUP_NAME                                             = 3887;
+exports.ER_DA_SSL_LIBRARY_ERROR                                                          = 3888;
+exports.ER_SECONDARY_ENGINE                                                              = 3889;
+exports.ER_SECONDARY_ENGINE_DDL                                                          = 3890;
+exports.ER_INCORRECT_CURRENT_PASSWORD                                                    = 3891;
+exports.ER_MISSING_CURRENT_PASSWORD                                                      = 3892;
+exports.ER_CURRENT_PASSWORD_NOT_REQUIRED                                                 = 3893;
+exports.ER_PASSWORD_CANNOT_BE_RETAINED_ON_PLUGIN_CHANGE                                  = 3894;
+exports.ER_CURRENT_PASSWORD_CANNOT_BE_RETAINED                                           = 3895;
+exports.ER_PARTIAL_REVOKES_EXIST                                                         = 3896;
+exports.ER_CANNOT_GRANT_SYSTEM_PRIV_TO_MANDATORY_ROLE                                    = 3897;
+exports.ER_XA_REPLICATION_FILTERS                                                        = 3898;
+exports.ER_UNSUPPORTED_SQL_MODE                                                          = 3899;
+exports.ER_REGEXP_INVALID_FLAG                                                           = 3900;
+exports.ER_PARTIAL_REVOKE_AND_DB_GRANT_BOTH_EXISTS                                       = 3901;
+exports.ER_UNIT_NOT_FOUND                                                                = 3902;
+exports.ER_INVALID_JSON_VALUE_FOR_FUNC_INDEX                                             = 3903;
+exports.ER_JSON_VALUE_OUT_OF_RANGE_FOR_FUNC_INDEX                                        = 3904;
+exports.ER_EXCEEDED_MV_KEYS_NUM                                                          = 3905;
+exports.ER_EXCEEDED_MV_KEYS_SPACE                                                        = 3906;
+exports.ER_FUNCTIONAL_INDEX_DATA_IS_TOO_LONG                                             = 3907;
+exports.ER_WRONG_MVI_VALUE                                                               = 3908;
+exports.ER_WARN_FUNC_INDEX_NOT_APPLICABLE                                                = 3909;
+exports.ER_GRP_RPL_UDF_ERROR                                                             = 3910;
+exports.ER_UPDATE_GTID_PURGED_WITH_GR                                                    = 3911;
+exports.ER_GROUPING_ON_TIMESTAMP_IN_DST                                                  = 3912;
+exports.ER_TABLE_NAME_CAUSES_TOO_LONG_PATH                                               = 3913;
+exports.ER_AUDIT_LOG_INSUFFICIENT_PRIVILEGE                                              = 3914;
+exports.OBSOLETE_ER_AUDIT_LOG_PASSWORD_HAS_BEEN_COPIED                                   = 3915;
```

Comment:
Shouldn't there not still be keys that are starting with `OBSOLETE_` in here?

Reply:
Ah, these were marked as obsolete before 8 was released, but didn't exist in 5 so I didn't think it mattered. But! I've made the tool now auto-remove the `OBSOLETE_` prefix if found!